config/locales/client.en.yml

```diff
@@ -3545,6 +3545,14 @@ en:
         delete: Permenantly Delete
         not_shown_again: This key will not be displayed again. Make sure you take a copy before continuing.
         continue: Continue
+        use_scopes: Use Scopes
```

Comment:
Can we flip this round, so that it sounds a bit more scary. Maybe "Global Key (allows all actions)" with a checkbox?